proxy/src/proxy.rs

```diff
@@ -89,9 +89,9 @@ pub async fn task_main(
 
                 handle_client(config, &cancel_map, session_id, socket).await
             }
-            .unwrap_or_else(|e| {
+            .unwrap_or_else(move |e| {
                 // Acknowledge that the task has finished with an error.
-                error!("per-client task finished with an error: {e:#}");
+                error!("per-client task session_id={session_id:?} finished with an error: {e:#}");
```

Comment:
`error!(?session_id, "...")`

Reply:
Good, didn't know that syntax. It is slightly weird that kv is rendered after the message, but anyway.